package.json

```diff
@@ -21,10 +21,15 @@
 	},
 	"homepage": "https://github.com/LambdaSchool/Server-Testing#readme",
 	"devDependencies": {
+		"cross-env": "^5.2.0",
```

Comment:
note that because you're not _using_ cross-env in your test script up above to control the DB_ENV variable, you're actually running all your tests on the "actual" database and never add anything to your test db

Reply:
Ahhhh, thank you, I must have missed that part and I have been trying to figure out what I'm doing wrong with regards to environment. 

Thank you!